vuepress/.gitignore

```diff
@@ -21,3 +21,5 @@ yarn-error.log*
 *.sw?
 /docs/.vuepress/dist/
 *.bak
+
+/.vscode
```

Comment:
I thought you'd need an ending slash here, so /.vscode/ and then delete the settings.json from the commit as well.

Reply:
@nathan please confirm how to delete settings.json from the commit?

Reply:
Discussed on call today